src/fluent-bit.c

```diff
@@ -1128,6 +1131,12 @@ int flb_main(int argc, char **argv)
                 flb_cf_section_property_add(cf_opts, s->properties, "match", 0, optarg, 0);
             }
             break;
+#ifdef FLB_SYSTEM_WINDOWS
+        case 'M':
+            flb_cf_section_property_add(cf_opts, service->properties,
+                                        "max_stdio", 0, optarg, 0);
```

Comment:
name has an underscore

Reply:
Fixed in https://github.com/fluent/fluent-bit/pull/9707/commits/beff608dc69d81aab78a03eae041b25438783efe